lib/index.ts

```diff
@@ -39,7 +64,11 @@ export async function inspect(
 
   const mvnArgs = buildArgs(targetFile, options.args);
   try {
```

Comment:
In the catch, can you add a check to show a different error based on what command has been run?

Reply:
and also a test for this case? 🙏

Reply:
added and added